### ISSUE TITLE
Android TF Classify Multi-label output example?

### ISSUE THREAD

**Issue by angels115** [tensorflow/tensorflow#11194]
What would you have to change to the existing tensorflow android TFClassify app demo to output multiple labels?  

For instance I have a multi output model trained in keras with multiple softmax output nodes. The current TFClassify example has one output node so would the solution be to merely add additional output nodes in the java files?

**Comment by aselle** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
